src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +52,12 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <IconChecklist
+                width={32}
+                height={32}
+                color={isAnon ? COLORS.anon_primary : COLORS.signed_primary}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change to conditionally render different icon colors based on the `isAnon` prop is a good approach to differentiate between anonymous and non-anonymous users. However, similar to the previous comment, please ensure that these colors are accessible and provide enough contrast for all users, including those with visual impairments.



<!-- This is an auto-generated comment by OSS CodeRabbit -->